packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -136,26 +133,30 @@ it('Create a directly funded channel between two wallets ', async () => {
   expect(getChannelResultFor(channelId, preFundA.channelResults)).toMatchObject({
     status: 'opening',
     turnNum: 0,
+    fundingStatus: 'Uncategorized',
   });
 
   const resultB0 = await b.pushMessage(getPayloadFor(participantB.participantId, preFundA.outbox));
 
   expect(getChannelResultFor(channelId, resultB0.channelResults)).toMatchObject({
     status: 'proposed',
     turnNum: 0,
+    fundingStatus: 'Uncategorized',
   });
 
   const prefundB = await b.joinChannel({channelId});
   expect(getChannelResultFor(channelId, [prefundB.channelResult])).toMatchObject({
     status: 'opening',
     turnNum: 1,
+    fundingStatus: 'Uncategorized',
```

Comment:
Why isn't this `ReadyToFund`? It's not its turn?

Reply:
`ReadyToFund` === all prefund states singed + all deposits before me have been made. I should probably document what each `FundingStatus` means.